src/main.rs

```diff
@@ -64,7 +65,10 @@ const RUSTC_SHIM_ENV_VAR_NAME: &'static str = "RLS_RUSTC_SHIM";
 type Span = span::Span<span::ZeroIndexed>;
 
 pub fn main() {
-    env_logger::init().unwrap();
+    if let Err(e) = log4rs::init_file("/home/jbushart/projects/rls/src/log4rs.yaml", Default::default()) {
```

Comment:
@jonasbb I think you forgot about removing this :smile:

Reply:
@Xanewok I am fully aware this is still in there. It is in the todo list I mentioned in the last comment "a quick rebase to remove commits from the history"

Reply:
oh okay, misunderstood that, sorry!